### DISCUSSION TITLE
The Age Docker Hub is said to be offering the latest version  We are providing the pre-release(1.5.0) version, not the latest version.

### DISCUSSION THREAD

**Discussion by shinhanbyeol** [apache/age#1520] [Q&A] [answered]
![image](https://github.com/apache/age/assets/76985229/7dcd44a5-aa4b-4761-84b3-17410fb7f9d8)
![image](https://github.com/apache/age/assets/76985229/5440c6ee-5620-40c9-b56a-2f1eb64a6fa9)
![image](https://github.com/apache/age/assets/76985229/549f7bc5-9fc5-4b98-9f1d-4e4b7c19fd06)

Should I watch the latest version as a pre-release tag or a latest?

**Comment by jrgemignani** [comment, marked as answer]
@shinhanbyeol On DockerHub, all of the `*_latest` are the latest for that branch. It is not related to the latest released version. It is the latest code for that branch.

Hope that is helpful.

**Reply by shinhanbyeol** [reply]
Thank you for answer!!

**Comment by jrgemignani** [comment]
@shinhanbyeol If this answers your question, please consider closing the ticket.
